src/inputs/labeled-inputs.scss

```diff
@@ -229,6 +230,31 @@
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
   }
+
+  &.iui-actionable {
+    align-items: center;
+    height: 90%;
+    width: fit-content;
```

Comment:
`width: fit-content` does not work in IE.  I know we are planning to phase out IE support eventually.  Does this have any negative effect on IE or will they just not see the animation?

Reply:
It is surprisingly functional in IE 😮
![IE](https://user-images.githubusercontent.com/9084735/135527422-e2ff9b3a-2ab6-44e4-a570-7ee1725aa2d6.gif)

Reply:
Turns out that's because `width: fit-content` is not even needed. So I've removed it